.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "workbench.colorCustomizations": {
+    "activityBar.background": "#2D3010",
+    "titleBar.activeBackground": "#3F4317",
+    "titleBar.activeForeground": "#FAFBF2"
+  }
+}
```

Comment:
This is not part of the rubocop setup, and I wonder if we can get rid of it? 😄 

I mean, this are editor preferences, and aren't needed to being able to run this project 🙌🏽 

WDYT?